datus/tools/bi_tools/registry.py

```
@@ -0,0 +1,121 @@
+        display_name: Optional[str] = None,
+    ) -> None:
+        self.platform = platform
+        self.adaptor_class = adaptor_class
+        self.auth_type = auth_type
+        self.display_name = display_name or platform.capitalize()
+
+
+class BIAdaptorRegistry:
+    """Central registry for BI adaptors."""
+
+    _adaptors: Dict[str, Type[BIAdaptorBase]] = {}
```

Ruff (0.14.10)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)